packages/common/src/base64.js

```diff
@@ -20,10 +19,7 @@ export function fromBase64url(str) {
  * @returns {string}
  */
 export function toBase64Url(str) {
-  return btoa(str)
-    .replace(/\+/g, `-`)
-    .replace(/\//g, `_`)
-    .replace(/\=/g, ``);
+  return UrlSafeBase64.encode(new Buffer(str));
```

Comment:
Doesn't this also need to call toString()? or does encode already return a string?